### DISCUSSION TITLE
Supporting  <details><summary> tag in HTML to create collapsible content？

### DISCUSSION THREAD

**Discussion by YiLee01** [gonzalezreal/swift-markdown-ui#239] [Q&A]
I am one of your users who has been using your iOS markdown parsing library. I am very grateful for your work, which has enabled me to easily edit and read markdown documents on my iOS device.

I noticed that in my markdown document, I used the <details><summary> tag in HTML to create collapsible content. However, I found that your parsing library does not seem to support this feature. Therefore, I would like to request your help.

Could you please add support for the <details><summary> tag in your parsing library? This way, I can use this feature in my markdown document and have it displayed correctly in your parsing library.

**Comment by gonzalezreal** [comment]
Hi @YiLee01,

I've been considering adding support for rendering some HTML blocks. It's a feature I would like to add at some point. But it is a lot of work, and I can't promise anything now.

**Comment by laike9m** [comment]
+1, I thought this is supported since it's part of GFM.

**Reply by gajddo00** [reply]
+1 currently `<sup>` is breaking our markdown text. I found an old closed [PR](https://github.com/gonzalezreal/swift-markdown-ui/pull/171), @gonzalezreal what was the challenge that made you close it and not add support for HTML tags? Maybe some of us can help? Any help is appreciated 🙏🏻

**Reply by laike9m** [reply]
Support `<sup>` would be great!
https://gist.github.com/wildmichael/5064046
